app/lib/constants/role.rb

```diff
@@ -18,6 +18,7 @@ module Role
                      "Resource Admin: Config",
                      "Resource Admin: Broadcast",
                      "Resource Admin: HtmlVariant",
-                     "Resource Admin: DisplayAd"].freeze
+                     "Resource Admin: DisplayAd",
+                     "Resource Admin: ListingCategory"].freeze
```

Comment:
I am not sure if it should be called Listing or ListingCategory

Reply:
as it's a category, I think `ListingCategory` is correct